test/dotnet.Tests/CommandTests/Pack/PackTests.cs

```diff
@@ -289,5 +290,117 @@ public void DotnetPackAcceptsRuntimeOption()
+            var result = new DotnetPackCommand(Log)
+                .WithWorkingDirectory(testInstance.Path)
+                .Execute(nuspecPath, "--property", "id=CustomID");
+
+            result.Should().Pass();
+
+            var outputDir = new DirectoryInfo(testInstance.Path);
+            outputDir.Should().Exist()
+                .And.HaveFile("PackNoCsproj.1.0.0.nupkg");
+        }
+
+        [Fact]
+        public void DotnetPack_AcceptsVersionOption()
+        {
+            var testInstance = _testAssetsManager.CopyTestAsset("TestNuspecProject")
+                .WithSource();
+            string nuspecPath = Path.Combine(testInstance.Path, "PackNoCsproj.nuspec");
+            var result = new DotnetPackCommand(Log)
+                .WithWorkingDirectory(testInstance.Path)
+                .Execute(nuspecPath, "--version", "1.2.3");
+
+            result.Should().Pass();
+
+            var outputDir = new DirectoryInfo(testInstance.Path);
+            outputDir.Should().Exist()
+                .And.HaveFile("PackNoCsproj.1.2.3.nupkg");
+        }
+
+        [Fact]
+        public void DotnetPack_FailsWhenVersionOptionHasNoValue()
+        {
+            var testInstance = _testAssetsManager.CopyTestAsset("TestNuspecProject")
+                .WithSource();
+            string nuspecPath = Path.Combine(testInstance.Path, "PackNoCsproj.nuspec");
+            var result = new DotnetPackCommand(Log)
+                .WithWorkingDirectory(testInstance.Path)
+                .Execute(nuspecPath, "--version");
+
+            result.Should().Fail();
+            result.StdErr.Should().Contain("Required argument missing for option: '--version'.");
+        }
+
+        [Fact]
+        public void DotnetPack_AcceptsCustomProperties()
+        {
+            var testInstance = _testAssetsManager.CopyTestAsset("TestNuspecProject")
+                .WithSource();
+
+            string nuspecPath = Path.Combine(testInstance.Path, "PackNoCsproj.nuspec");
+
+            var result = new DotnetPackCommand(Log)
+                .WithWorkingDirectory(testInstance.Path)
+                .Execute(
+                nuspecPath, "--property", "id=CustomValue",
+                "--property", "authors=CustomAuthor"
+                );
+
+            result.Should().Pass();
+        }
+        [Fact]
+        public void DotnetPack_AcceptsConfigurationOption()
+        {
+            var testInstance = _testAssetsManager.CopyTestAsset("TestNuspecProject")
+                .WithSource();
+            string nuspecPath = Path.Combine(testInstance.Path, "PackNoCsproj.nuspec");
+            var result = new DotnetPackCommand(Log)
+                .WithWorkingDirectory(testInstance.Path)
+                .Execute(nuspecPath, "--configuration", "Release");
+
+            result.Should().Pass();
+            var outputDir = new DirectoryInfo(testInstance.Path);
+            outputDir.Should().Exist()
```

Comment:
Ensure the option is actually imapcting the end result by opening the nuspec and checking.